docs/Customizing-your-instance-of-cBioPortal.md

```diff
@@ -257,3 +257,8 @@ skin.documentation.markdown=false
 
 If your documentation contains a relative link, cBioPortal will assume it uses the same format and, when clicked, will try to interpret the page as such. If your documentation contains an absolute link, cBioPortal will open it in a new window when clicked.
 Please be aware that the links may be case-sensitive! E.g. https://github.com/cBioPortal/cbioportal/wiki/News.md is not the same as https://github.com/cBioPortal/cbioportal/wiki/news.md
+
+# Custom styling of the patient view's clinical data
+The [Patient View](http://www.cbioportal.org/case.do?cancer_study_id=lgg_ucsf_2014&case_id=P04) shows several [clinical attributes](File-Formats.md#clinical-data) at the top of the page, e.g. `AGE`, `SEX`:
+![test](../test/end-to-end/screenshots/firefox/patient_view_lgg_ucsf_2014_case_id_P04.png)
+The order, styling and visibility of those [attributes](File-Formats.md#clinical-data) at the top can be changed by editing the [patient view's clinical attributes CSS file](../portal/src/main/webapp/css/patient-view/clinical-attributes.css).
```

Comment:
Just a thought: might it be a sensible compromise to look for the instance-specific file in one set place, and if a file by that name exists, use it instead of the default `portal/src/main/webapp/css/patient-view/clinical-attributes.css`?